ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/Help.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.cli.console.cmd;
+
+import io.confluent.ksql.cli.console.Console;
+import java.util.Objects;
+
+class Help implements CliSpecificCommand {
+
+  private final Console console;
+
+  Help(final Console console) {
+    this.console = Objects.requireNonNull(console, "console");
+  }
+
+  @Override
+  public String getName() {
+    return "help";
+  }
+
+  @Override
+  public void printHelp() {
+    console.writer().println("help:");
```

Comment:
Not really part of this PR, but i wish we didn't have to do `console.writer().println(...)`...

Reply:
Agreed